src/Raven.Server/Documents/Indexes/Auto/AutoMapIndex.cs

```diff
@@ -17,6 +18,14 @@ private AutoMapIndex(AutoMapIndexDefinition definition)
 
         public static AutoMapIndex CreateNew(AutoMapIndexDefinition definition, DocumentDatabase documentDatabase)
         {
+            var searchEngineType = MapIndexDefinition.GetSearchEngineType(null, documentDatabase.Configuration);
+            if (searchEngineType is SearchEngineType.Corax)
+            {
+                //Since definition is 
+                foreach (var field in definition.IndexFields.Values)
```

Comment:
Would be better to do `__all_fields`, no?

Reply:
I will check that, in `static indexes` context we want to scan user config and throw (with explanation) when a user set it to `No`

Reply:
Ok, I changed the approach after the conversation with @arekpalinski, more info is in the description of PR.